src/main/proto/wfa/virtual_people/core/labeler/input_labeler.proto

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa_virtual_people;
+
+option java_package = "org.wfanet.virtualpeople.core.labeler";
+option java_multiple_files = true;
+
+import "wfa/virtual_people/common/event.proto";
+import "wfa/virtual_people/common/label.proto";
+import "wfa/virtual_people/common/model.proto";
+
+// Request to Label Events
+message LabelEventsRequest {
+  CompiledNode root_node = 1;  // TODO: change this to repeated nodes
```

Comment:
I heard that a model file can be as big as several giga bytes. Not show how big the resulted compiledNode proto object can be. In other words, not sure if it is a good idea to call the label via JNI. Even if using JNI, we may want to share the memory between JVM and C++

Reply:
I agree on this one 100%